web/src/components/Header/HeaderLeftPanel.tsx

```diff
@@ -112,6 +116,44 @@ const HeaderLeftPanel: React.FC<HeaderLeftPanelProps> = ({
   useOnClickOutside(ref, () => setOpenEntryPointPicker(false))
   const [openEntryPointPicker, setOpenEntryPointPicker] = useState(false)
 
+  const [numOpsToFetch, setNumOpsToFetch] = useState(0)
+
+  // display syncing indicator when numOpsToFetch > 0
+  useEffect(() => {
+    const fetchOpData = async () => {
+      const adminWs = await getAdminWs()
+      const dnas = await adminWs.listDnas()
```

Comment:
I'd like to cache the result of this call in state, and make that state a dependency of this useEffect. 
So it would occur in a separate useEffect

Reply:
sounds good

Reply:
i am struggling to understand your request exactly. so i have extracted `fetchOpData` into a named async function, returning `Promise.resolve(sum)` and moved the function outside the `useEffect`.

so my interpretation of your request is to do something like
```
const interval = setInterval(()=> {
  const numOpsToFetch = await fetchOpData()
  setNumOpsToFetch(numOpsToFetch)
}, 5000)
```
inside of this current `useEffect`. You mentioned extracting it to a separate one. can you expand on this a little? and why it would be beneficial?

Reply:
I want to be able to call the networkInfo more regularly than the 5 seconds, but I don’t want to have listDnas be called every time also